packages/kotti-ui/source/kotti-field-file-upload/components/FileItemRemote.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-	<ItemLayout :name="fileInfo.name">
+	<ItemLayout
+		:class="wrapperClasses"
```

Comment:
Shouldn't the `ItemLayout` handle its own styling? It is already accepting the new `isInternal` flag which is what these wrapper classes depend on.